psalm.xml

```diff
@@ -6,55 +6,63 @@
         </ignoreFiles>
     </projectFiles>
 
     <issueHandlers>
         <UndefinedClass>
             <errorLevel type="suppress">
                 <!-- zendframework/zend-server is not a direct package nor build dependency, and it should stay that way -->
+                <referencedClass name="Zend\Http\Client\Adapter\Exception\RuntimeException"/>
                 <referencedClass name="Zend\Server\Client"/>
+                <referencedClass name="Zend\XmlRpc\Server"/>
+                <referencedClass name="Zend\XmlRpc\Client"/>
             </errorLevel>
         </UndefinedClass>
-        <LessSpecificReturnType errorLevel="info" />
+        <LessSpecificReturnType errorLevel="info"/>
 
         <!-- level 3 issues - slightly lazy code writing, but provably low false-negatives -->
 
-        <DeprecatedMethod errorLevel="suppress" />
-        <DeprecatedProperty errorLevel="suppress" />
-        <DeprecatedClass errorLevel="suppress" />
-        <DeprecatedConstant errorLevel="suppress" />
-        <DeprecatedInterface errorLevel="suppress" />
-        <DeprecatedTrait errorLevel="suppress" />
+        <DeprecatedMethod errorLevel="suppress"/>
+        <DeprecatedProperty errorLevel="suppress"/>
+        <DeprecatedClass errorLevel="suppress"/>
+        <DeprecatedConstant errorLevel="suppress"/>
+        <DeprecatedInterface errorLevel="suppress"/>
+        <DeprecatedTrait errorLevel="suppress"/>
 
-        <InternalMethod errorLevel="suppress" />
-        <InternalProperty errorLevel="suppress" />
-        <InternalClass errorLevel="suppress" />
+        <ForbiddenCode errorLevel="suppress"/>
 
-        <MissingClosureReturnType errorLevel="info" />
-        <MissingReturnType errorLevel="info" />
-        <MissingPropertyType errorLevel="error" />
-        <InvalidDocblock errorLevel="error" />
-        <MisplacedRequiredParam errorLevel="error" />
+        <InternalMethod errorLevel="suppress"/>
+        <InternalProperty errorLevel="suppress"/>
+        <InternalClass errorLevel="suppress"/>
 
-        <PropertyNotSetInConstructor errorLevel="suppress" />
-        <MissingConstructor errorLevel="error" />
-        <MissingClosureParamType errorLevel="info" />
-        <MissingParamType errorLevel="info" />
+        <MissingClosureReturnType errorLevel="info"/>
+        <MissingReturnType errorLevel="info"/>
+        <MissingPropertyType errorLevel="error"/>
+        <InvalidDocblock errorLevel="error"/>
+        <MisplacedRequiredParam errorLevel="error"/>
 
-        <RedundantCondition errorLevel="suppress" />
+        <PropertyNotSetInConstructor errorLevel="suppress"/>
+        <MissingConstructor errorLevel="error"/>
+        <MissingClosureParamType errorLevel="info"/>
+        <MissingParamType errorLevel="info"/>
 
-        <DocblockTypeContradiction errorLevel="error" />
-        <RedundantConditionGivenDocblockType errorLevel="error" />
+        <RedundantCondition errorLevel="suppress"/>
```

Comment:
Is this emitted somewhere erroneously? It's the only issue here that should never be emitted for good code (so if it is, that's a Psalm bug)

Reply:
Will need to inspect this: I plan to raise the levels before merging

Reply:
Bumped all inspection levels